Feat(ERF-56): Sharing overlay

vogelino opened the pull request:
This PR introduces the sharing overlay

lucasoeth reviewed (approved):
Great addition. Thanks to your page infrastructure we were able to add this good feature in this sprint! Code wise everything looks clean :)

A style suggestion I have is maybe invert the colors of the botton when the link is copied. Only if you think it looks good tho
Secondly the Icon in the share button is not centered on mobile (prob because of the grid stuff)
Lastly, the copy function isn't working on mobile either. Not sure if you tested it on your phone but this might be an ios version thing again.

Otherwise, feel free to merge it since most of these should pop up during the user testing phase aswell

vogelino commented:
> A style suggestion I have is maybe to invert the colors of the button when the link is copied. Only if you think it looks good tho

I agree it looks good. I did change it.

> Secondly the Icon in the share button is not centered on mobile (prob because of the grid stuff)

I fixed it using text-align center and some padding

> Lastly, the copy function isn't working on mobile either. Not sure if you tested it on your phone but this might be an ios version thing again.

I did try it on my phone and it worked indeed but I added a fallback for older versions now.
